index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="style.css">
+	<title>Gleaming Gemstones</title>
+
+</head>
+
+<body>
+	<!--hero section-->
+	<div class="hero-video">
+		<video autoplay muted loop>
+			<source src="3633003-hd_1906_1080_30fps.mp4" type="video/mp4">
+			Din webbläsare stödjer inte HTML5 video.
```

Comment:
code is in general clear and well structured, but could be good to be consistent in english